regulations/static/regulations/js/source/views/comment/comment-confirm-view.js

```diff
@@ -29,8 +29,8 @@ var CommentConfirmView = Backbone.View.extend({
     this.interval = window.setInterval(
       function() {
         $.getJSON(url).then(function(resp) {
-          this.setPdfUrl(resp.pdfUrl);
           if (resp.trackingNumber) {
+            this.setPdfUrl(resp.pdfUrl);
```

Comment:
This was the very issue we discussed in Slack a couple of days back:

> 
Vraj Mohan (PHL) [11:27 AM] 
Would we want to allow the user to download the final comment PDF (the one ​_without_​ the draft watermark), if the submission to regulations.gov is not successful (or is still in process)? I think not. Does anyone think differently?

> 
CM Lubinski [11:29 AM] 
hrm. I think the argument might be "allow the user to submit it via another means", but we say we'll be doing that, so I think you are correct